ci-templates/README.md

```diff
@@ -0,0 +1,8 @@
+<!--
+SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+SPDX-License-Identifier: Apache-2.0
+-->
+
+Currently, we support the following instances:
+
+- Gitlab CI/CD ([More information](./gitlab/README.md))
```

Comment:
```suggestion
# Supported instances

- Gitlab CI/CD ([More information](./gitlab/README.md))
```
I'm not sure which variant is better, I just wanted to throw that in here. So choose whichever variant you like better.